tests/test_invalid_jaxlib.py

```diff
@@ -3,14 +3,25 @@
 import importlib
 
 
-def test_raise_on_outdated_jaxlib(monkeypatch):
+@pytest.mark.parametrize("fake_jaxlib_version", ["0.0.0", "0.1.61", "0.1.61+cuda110"])
```

Comment:
```suggestion
@pytest.mark.parametrize("fake_jaxlib_version", ["0.0.0", "0.1.61", "0.1.61+cuda110", "0.1.61a1"])
```

Reply:
would it work with this? it's also a standard python version identifier

Reply:
Now it does :)